apollo-runtime/src/main/java/com/apollographql/apollo/ApolloMutationCall.java

```diff
@@ -0,0 +1,44 @@
+package com.apollographql.apollo;
+
+import com.apollographql.apollo.api.Mutation;
+import com.apollographql.apollo.api.OperationName;
+import com.apollographql.apollo.cache.CacheHeaders;
+import com.apollographql.apollo.cache.normalized.CacheControl;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A call prepared to execute GraphQL mutation operation.
+ */
+public interface ApolloMutationCall<T> extends ApolloCall<T> {
+
+  /**
+   * <p>Sets a list of GraphQL query names to be re-fetched once this mutation completed.</p>
+   * In order to get queries to be re-fetched you must obtain {@link ApolloQueryWatcher} from provided list of
+   * queries before running this mutation.
+   *
+   * @param operationNames array of {@link OperationName} query names to be re-fetched
+   * @return {@link ApolloMutationCall} that will trigger re-fetching provided queries
+   */
+  @Nonnull ApolloMutationCall<T> refetchQueries(@Nonnull OperationName... operationNames);
+
+  @Nonnull @Override ApolloMutationCall<T> cacheControl(@Nonnull CacheControl cacheControl);
```

Comment:
Yes, cacheControl does not make sense for mutations. It will always be NETWORK_ONLY. If we write to the cache can be controlled by `cacheHeaders`. It's generally important to write mutations results to the cache, as this is what triggers updates to `apolloWatchers`.